charts/cluster-api-runtime-extensions-nutanix/templates/nfd/manifests/helm-addon-installation.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
   name: '{{ .Values.hooks.nfd.helmAddonStrategy.defaultValueTemplateConfigMap.name }}'
 data:
   values.yaml: |-
+    image:
+      tag: v0.15.2-minimal
```

Comment:
Do we get a variable with the chart version? Or can automatically set this when generating these values?
We will forget to change this.

Reply:
i'll be following up with a tool to sync these files with. this was just one i noticed that was broken right now 

hack/addons/kustomize/**/helm-values.yaml

Reply:
Merge this and follow up in a different PR, i guess that's what you're saying @faiq ?